### PR TITLE
Add expertise table

### PR DESCRIPTION
**Pull request by LaurieLinz**
Added and joined table.  Tests are green

### CODE REVIEW

**Comment by rylanb** [comment]
I'd like to see a test for expertise like the spec/features/creating_contacts_spec.rb test. For consistency. There is: spec/requests/expertises_spec.rb, so guess it depends which way you want to go with tests.

Also, the contacts db model seems to have the old name 'service' and not the 'met_on' version, which breaks the form there. 

Taking out the controller tests and 'spec/views/contacts/index.html.erb_spec.rb' like tests to get tests passing was done why?

**Comment by rylanb** [comment]
Otherwise looks and works well for expertises! Nice work on the new look/feel, too. Make sure to update links in the navbar as they don't seem to be working, either.

File: app/views/contacts/_form.html.erb [hunk 1]
```diff
@@ -40,13 +40,23 @@
     <%= f.text_field :availability %>
   </div>
   <div class="field">
-    <%= f.label :skillset %> 1 for junior, 2 for mid-level, 3 for expert 4 for overconfident! <br>
-    <%= f.number_field :skillset %>
+    <%= f.label :skillset %> <br>
+    <%= f.select :skillset, options_for_select(Contact.skillsets.keys, @contact.skillset) %>
```
Comment (rylanb) [inline app/views/contacts/_form.html.erb:44]: Definitely should add classes from Bootstrap for these things and form-group containers to space things out.

Not urgent. Nor required.

File: app/views/expertises/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <tbody>
+      <%= content_tag_for(:tr, @expertises) do |expertise| %>
+            <td><%= expertise.discipline %></td>
+            <td><%= link_to 'Show', expertise %></td>
+        <td><%= link_to 'Edit', edit_expertise_path(expertise) %></td>
+        <td><%= link_to 'Destroy', expertise, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```
Comment (rylanb) [inline app/views/expertises/index.html.erb:29]: How do you get to expertises currently?

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+  resources :expertises
   resources :contacts
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'contacts#index'
```
Comment (rylanb) [inline config/routes.rb:8]: 👍

File: spec/models/contact_spec.rb [hunk 4]
```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Contact, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
```
Comment (rylanb) [inline spec/models/contact_spec.rb:5]: This should stay as a unit test for the future when you have required fields (which there are some and should be added as validates_presence_of and then tested here.

File: spec/views/contacts/edit.html.erb_spec.rb [hunk 5]
```diff
@@ -1,39 +0,0 @@
...
-      assert_select "input#contact_service[name=?]", "contact[service]"
-
-      assert_select "input#contact_availability[name=?]", "contact[availability]"
-
-      assert_select "input#contact_skillset[name=?]", "contact[skillset]"
-
-      assert_select "textarea#contact_notes[name=?]", "contact[notes]"
-    end
-  end
-end
```
Comment (rylanb) [inline spec/views/contacts/edit.html.erb_spec.rb:39]: This should still be in here...

File: app/views/expertises/index.html.erb (hunk 2 above)
Comment (LaurieLinz) [inline app/views/expertises/index.html.erb:29]: I just go there locally.  I will make a button.

File: app/views/contacts/_form.html.erb (hunk 1 above)
Comment (LaurieLinz) [inline app/views/contacts/_form.html.erb:44]: I can do this.

File: spec/views/contacts/edit.html.erb_spec.rb (hunk 5 above)
Comment (LaurieLinz) [inline spec/views/contacts/edit.html.erb_spec.rb:39]: I will put them back

**Pull request merged**